en/docs/apis/api-concepts.md

```diff
@@ -1,23 +1,32 @@
 # API Concepts
-Before you start managing APIs via Choreo, there are basic concepts that you need to understand. This section walks you through all the concepts you need to know. 
+Choreo APIs allow you to create APIs, manage APIs, and expose services and external APIs as managed APIs. Here are some basic concepts which will help you understand Choreo APIs as you create and manage your APIs in the [Choreo Console]({{choreo_console}}).
```

Comment:
"Choreo APIs " here does not sound right. Shall we say "Choreo’s API management capabilities allow . . ."

Reply:
+1